### DISCUSSION TITLE
Customize strokeWidth

### DISCUSSION THREAD

**Discussion by LoiKos** [airbnb/visx#1205] [Q&A]
Hi, 
I'm using '@visx/xychart' in version `"^1.10.0"` and i was trying to set custom tick width but i didn't make it. 

I tried by adding strokeWidth in theme like : 

```js
const theme = buildChartTheme({
  backgroundColor: '#fff',
  colors: [
    hexRgb('#4580E6', { format: 'css', alpha: 0.6 }),
    hexRgb('#15B371', { format: 'css', alpha: 0.6 }),
    hexRgb('#15B371', { format: 'css', alpha: 0.6 }),
    hexRgb('#F29D49', { format: 'css', alpha: 0.6 }),
    hexRgb('#F29D49', { format: 'css', alpha: 0.6 }),
    '#182026',
  ],
  gridColor: '#30404D',
  gridColorDark: '#30404D',
  tickLength: 8,
  xTickLineStyles: { strokeWidth: 2 }, // Not working
}) 
```

Should i use a custom tick component to do that ?

CodeSandbox : https://codesandbox.io/s/blazing-snow-mvsuq?file=/src/App.tsx

**Comment by williaster** [comment]
Hey @LoiKos great question! I was just looking into why this doesn't work and surprisingly we don't actually support overriding this any way except with `className`/old-school css (see this [modified sandbox](https://codesandbox.io/s/infallible-dan-crp1r)). 

![image](https://user-images.githubusercontent.com/4496521/118062002-e8d29a00-b34a-11eb-96e0-5378d8ce6a3b.png)


[This](https://github.com/airbnb/visx/blob/master/packages/visx-axis/src/axis/Ticks.tsx#L37) is the relevant line of code that would need to take in the `tickLineProps`, if you or anyone else is interested in adding this to `@visx/axis` it would then be useable through `@visx/xychart` as well.

**Reply by LoiKos** [reply]
I have done a PR on that :  #1211
